easybuild/easyblocks/h/hadoop.py

```diff
@@ -93,6 +96,19 @@ def sanity_check_step(self):
         }
         super(EB_Hadoop, self).sanity_check_step(custom_paths=custom_paths)
 
+        fake_mod_data = self.load_fake_module(purge=True)
+        cmd = "hadoop checknative -a"
+        out, _ = run_cmd(cmd, log_all=True, simple=False, log_ok=True)
+        self.clean_up_fake_module(fake_mod_data)
+
+        not_found = []
+        for lib, lib_path  in self.cfg['extra_native_libs']:
+            if not re.search(r'%s: *true' % lib, out):
```

Comment:
We can check the path. We can run nm on the file and check for required symbols. How long is a piece of string?